cooperative_lanechange/config/parameters.yaml

```diff
@@ -58,7 +58,7 @@ curvature_moving_average_window_size : 9
 curvature_calc_lookahead_count : 1
 
 # Int: Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
-downsample_ratio : 8
+downsample_ratio : 1
```

Comment:
Is a 1:1 point downsample ratio really intended?

Reply:
yeah because sometimes in a small map like TFHRC, we run out of points if we sample it down.